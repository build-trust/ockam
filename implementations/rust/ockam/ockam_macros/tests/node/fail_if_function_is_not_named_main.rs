#[ockam_macros::node]
fn foo(mut c: ockam::Context) {
    c.stop().await.unwrap();
}
