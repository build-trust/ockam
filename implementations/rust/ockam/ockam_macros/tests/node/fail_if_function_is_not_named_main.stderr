error: The function name must be `main`
 --> $DIR/fail_if_function_is_not_named_main.rs:2:1
  |
2 | fn foo(mut c: ockam::Context) {
  | ^^

error[E0601]: `main` function not found in crate `$CRATE`
 --> $DIR/fail_if_function_is_not_named_main.rs:1:1
  |
1 | / #[ockam_macros::node]
2 | | fn foo(mut c: ockam::Context) {
3 | |     c.stop().await.unwrap();
4 | | }
  | |_^ consider adding a `main` function to `$DIR/tests/node/fail_if_function_is_not_named_main.rs`
